RELEASE_NOTES.md

```diff
@@ -96,6 +96,11 @@ v1.0.0-alpha.xx
   feature.
   [#1088](https://github.com/OpenAssetIO/OpenAssetIO/issues/1088)
 
+- Removed the unused `PythonPluginSystemManagerPlugin.uiDelegate`
```

Comment:
[consider] (not related to this line, just somewhere to put it)

Not sure how much we want to strip mentions of UI delegation in general. Some other relevant places that mention UI:

`openassetio/__init__.py`
```
- Replacement/augmentation of in-application UI elements such as
  browsers and other panels/controls.
```
`MainPage.dox`
```
 * - Replacement/augmentation of in-application UI elements such as
 *   browsers and other panels/controls.
```
`ForHosts.dox`
```
 * - Make the drawing of any parameters that may hold an @ref
 *   entity_reference delegatable to the @needsref ManagerUIDeleagate.
 [...]
 * @note The UI classes have not yet been migrated from the
 * `FnAssetAPI` code base.
```
`ForManagers.dox`
```
 * @subsection manager_todo_ui Embedding Custom UI Within the Host
 *
 * @note The UI layer of OpenAssetIO is yet to be ported to this code base,
 * this documentation section is currently in-progress.
```
`Glossary.dox`
```
 * It's worth noting however, that if no custom UI widgets are supplied
 * by a manager, a @ref host will simply display it as a standard
```

Reply:
Good call, thanks! Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/1152/commits/f1e31febb6e4241f08939127c10607cff95c6332)